### ISSUE TITLE
Interaction construction of and/or signifier in tag area of data tables

### ISSUE THREAD

**Issue by jenniferthibault** [18F/openFEC-web-app#1330]
![screen shot 2016-07-01 at 1 28 22 am](https://cloud.githubusercontent.com/assets/11636908/16512612/26afa7da-3f2b-11e6-8355-2a2adc535242.png)

![screen shot 2016-07-01 at 1 30 10 am](https://cloud.githubusercontent.com/assets/11636908/16512636/6579803a-3f2b-11e6-886f-5da3100f3158.png)

Design criteria for completion from https://github.com/18F/openFEC-web-app/issues/1303#issuecomment-229809078

- [ ] figure out the formula/visual expression of how to show x, y, and z for elements in the same or different categories (UX and content work)
- [ ] Make comprehensible sentence out of it: Viewing 13,890 candidates matching: `tag` and `tag` and `tag` (ux design & visual layout)

**Comment by onezerojeremy** [comment]
After a great meeting with @jeanninehunter & @emileighoutlaw today, landed on three tactics for perusal. The tactics are not mutually exclusive, and each comes with their own bag of trade-offs. If I'm summarizing our content trust correctly they were much in favor of explicit `and`s and `or`s, and on the fence about explicitly stating the parent category (as in tactic 2) in the filter bar. @jeanninehunter & @emileighoutlaw plunk what i'm forgetting or mis-summarizing here. 

### tactic 1: include `and` & `or` in the filter bar. Make them visually distinct. 
- 😄  by explicitly stating where filter applications are applied in an `and` & `or`, we keep the user in the loop of what the system is doing. 
- 😄 by differentiating `and` & `or` we make it somewhat easier to tell where the `or` borders are. 
- 😕  it's still a long string. 
- 😕  because the filters are all strung together, it's still a little hard to scan and see where one filter ends.
- 😕  probably an implementation headache

### tactic 2: `and` operator includes filter category
- 😄 `and` operator explicitly stating what it is makes it clearer what the filter is doing
- 😕  it's an even longer string.
- 😕 probably an implementation headache

### tactic 3: `and` operator links
- 😄 `and` operator explicitly shows what its boundaries are makes it clearer what the filter is doing
- 😕  it's an even longer string.
- 😕 woooo doggie that's ugly. what a visual pickle, especially on mobile. 
- 😕 probably an implementation headache

![and-or filters playground](https://cloud.githubusercontent.com/assets/474916/16786812/79acbad4-484c-11e6-86d6-17545a54cc94.png)

**Comment by onezerojeremy** [comment]
uploading the zipped sketch file now, in the event it's useful. 
[betaFEC-data-tables revised.sketch.zip](https://github.com/18F/openFEC-web-app/files/360437/betaFEC-data-tables.revised.sketch.zip)

**Comment by noahmanger** [comment]
Super interesting stuff. I think it's really helpful to include the category somewhere, as it gives the "ands" and "ors" more context. Interested in hearing any thoughts @xtine might have from the implementation angle. (I can also talk through more what this is all about).

**Comment by noahmanger** [comment]
Update: I showed this to my critique group, and the consensus was that option 3 worked the best, as it balanced being descriptive while avoiding being overly wordy, fwiw.

**Comment by emileighoutlaw** [comment]
Option 3 is my preference too! The visual groupings are so clear there, which is something I was really craving. Also, +1 to not too wordy.

**Comment by jenniferthibault** [comment]
+1 option 3. I'm about to kick myself once I try to style that.

**Comment by onezerojeremy** [comment]
😁 I have tremendous faith in ya.

Don't think you will, but: Don't feel obliged to interpret it too
literally!

**Comment by noahmanger** [comment]
Closing in favor of #1329 for visuals.
